### ISSUE TITLE
changes to scrollOffsetFn ignored

### ISSUE THREAD

**Issue by henrikq** [tannerlinsley/react-virtual#105]
I tried to implement something like:
```
  const rowVirtualizer = useVirtual({
    size,
    parentRef,
    estimateSize,
    scrollOffsetFn: () => {
      if (!parentRef.current) return 0;
      return Math.max(0, parentRef.current.scrollTop - offset);
    },
  });
```
and to my surprise it didn't work when offset was dynamic. It seems that initial `scrollOffsetFn` is used, and subsequent `scrollOffsetFn` are ignored.

I guess its as simple as needing `scrollOffsetFn` to be in the useMemo deps here:
https://github.com/tannerlinsley/react-virtual/blob/ea6b541a4e3d85d515c4d51af522643ec2be65c1/src/index.js#L100

**Comment by henrikq** [comment]
I managed to get it to work by calculating `offset` inline using refs.

Thanks for awesome packages.

**Comment by piecyk** [comment]
This should be fixed via #111

**Comment by piecyk** [comment]
@henrikq as this is now resolved, can you close the issue?
